contracts/deployables/freeze-voting/FreezeVotingAzoriusV1.sol

```diff
@@ -227,25 +231,20 @@ contract FreezeVotingAzoriusV1 is
         );
     }
 
-    // ======================================================================
-    // FreezeVotingBase
-    // ======================================================================
-
-    // --- State-Changing Functions ---
-
     /**
-     * @inheritdoc IFreezeVotingBase
-     * @dev Extends base unfreeze to also clear the freeze proposal strategy.
-     * This ensures a fresh strategy snapshot for the next freeze proposal.
+     * @inheritdoc IFreezeVotingAzoriusV1
      */
     function unfreeze() public virtual override onlyOwner {
+        FreezeVotingBaseStorage storage $base = _getFreezeVotingBaseStorage();
         FreezeVotingAzoriusStorage storage $ = _getFreezeVotingAzoriusStorage();
 
+        // Reset all freeze state
+        $base.isFrozen = false;
+        $base.freezeProposalCreated = 0;
+        $base.freezeProposalVoteCount = 0;
+
         // Clear the strategy snapshot to ensure fresh capture next time
         $.freezeProposalStrategy = address(0);
```

Comment:
Do we need this line?

Reply:
as we spoke about, we probably don't "need" it. but maybe it's "nice to have"?

if we removed it, that means that in the periods of time in which a freeze was _not_ in effect, this property would hold the strategy address that was used in the _last_ freeze, which is just stale.

i personally think it's worth the extra SSTORE (of the zero address, which is cheaper than a non-zero SSTORE) to clean up that variable.